### PR TITLE
Link via device on device page

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Small little details, linking the device it is connected via.

CleanShot 2022-02-05 at 00 00 38

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Review by zsarnett — DISMISSED** [review]

**Review by balloob — DISMISSED** [review]
nice!

File: src/panels/config/devices/device-detail/ha-device-info-card.ts [hunk 1]
```diff
@@ -135,6 +138,10 @@ export class HaDeviceCard extends LitElement {
         color: var(--secondary-text-color);
         word-wrap: break-word;
       }
+      a {
```
Comment (balloob) [inline src/panels/config/devices/device-detail/ha-device-info-card.ts:141]: Weird this was necessary

File: src/panels/config/devices/device-detail/ha-device-info-card.ts [hunk 2]
```diff
@@ -135,6 +138,10 @@ export class HaDeviceCard extends LitElement {
...
+        text-decoration: underline;
+        color: var(--primary-color);
+      }
```
Comment (bramkragten) [inline src/panels/config/devices/device-detail/ha-device-info-card.ts:144]:
```suggestion
```

Comment (bramkragten) [reply src/panels/config/devices/device-detail/ha-device-info-card.ts:144]: Use `haStyle`;

```
  static get styles(): CSSResultGroup {
    return [haStyle, css`
      :host {
        display: block;
      }
      ....
      `,
   ];
```

Comment (frenck) [reply src/panels/config/devices/device-detail/ha-device-info-card.ts:144]: This is needed, or else the link will be default dark blue.

File: src/panels/config/devices/device-detail/ha-device-info-card.ts (hunk 1 above)
Comment (frenck) [reply src/panels/config/devices/device-detail/ha-device-info-card.ts:141]: Dunno either, It looked weird/of without it.

File: src/panels/config/devices/device-detail/ha-device-info-card.ts (hunk 2 above)
Comment (frenck) [reply src/panels/config/devices/device-detail/ha-device-info-card.ts:144]: the text-decoration can go actually, I experimented doing it without, but that looked of as well.

Comment (frenck) [reply src/panels/config/devices/device-detail/ha-device-info-card.ts:144]: I now see your latest comment, had the tab open, don't know why GitHub didn't add that response (reload page revealed it). Will adjust.

File: src/panels/config/devices/device-detail/ha-device-info-card.ts [hunk 3]
```diff
@@ -112,29 +116,36 @@ export class HaDeviceCard extends LitElement {
...
+        }
+        .manuf,
+        .model {
+          color: var(--secondary-text-color);
+          word-wrap: break-word;
+        }
+        a {
+          text-decoration: underline;
+          color: var(--primary-color);
+        }
```
Comment (zsarnett) [inline src/panels/config/devices/device-detail/ha-device-info-card.ts:147]:
```suggestion
```

**Review by zsarnett — DISMISSED** [review]

File: src/panels/config/devices/device-detail/ha-device-info-card.ts (hunk 3 above)
Comment (frenck) [reply src/panels/config/devices/device-detail/ha-device-info-card.ts:147]: 🤦

**Review by zsarnett — APPROVED** [review]

**Pull request merged**